lib/ansible/modules/cloud/amazon/redshift_security_group.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+
+# (c) 2015, Jens Carl, Hothead Games Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+module: redshift_security_group
+author: '"Jens Carl (@j-carl), Hothead Games Inc."'
+short_description: Redshift security group management
+description:
+  - allows you to create, delete, and modify Redshift security groups
+version_added: "2.2"
+options:
+  state:
+    description:
+      - Create or delete the security group.
+    default: 'present'
+    choices: ['present', 'absent']
+  group_name:
+    description:
+      - Name of the security group.
+    required: false
+    default: 'Default'
+    aliases: ['name']
+  description:
+    description:
+      - Describe the security group.
+    required: true
+  rules:
+    description:
+      - List of rules to add to security group.
+    required: true
+
+extends_documentation_fragment:
+    - aws
+requirements: [ 'boto' ]
+'''
+
+EXAMPLES = '''
+# Note: None of these examples set aws_access_key, aws_secret_key, or region.
+# It is assumed that their matching environment variables are set.
+
+# Create a security group with CIDR rule.
+    redshift_security_group:
+        state: present
+        group_name: test_with_cidr
+        description: Security group with CIDR
+        rules:
+            - cidr: 0.0.0.0/0
+
+# Create a security group with EC2 security group
+    redshift_security_group:
+        state: present
+        group_name: test_with_ec2_sec_group
+        description: Security group with EC2 security group
+        rules:
+            - sg_name: ec2_sg_test
+              sg_owner: <AWS account id>
+
+# Add a CIDR rule to an existing EC2 rule.
+    redshift_security_group:
+        state: present
+        group_name: test_with_ec2_sec_group
+        description: Security group with EC2 security group and CIDR
+        rules:
+            - sg_name: ec2_sg_test
+              sg_owner: <AWS account id>
+            - cidr: 0.0.0.0/0
+
+# Delete a security group.
+    redshift_security_group:
+        state: absent
+        group_name: test_with_ec2_sec_group
+'''
+
+RETURN = '''
+changed:
+    description: A flag indicating if any change was made or not
+    returned: success
+    type: boolean
+    sample: True
+'''
+# import module snippets
```

Comment:
Module imports are best between the header comments and the docs, else flake8 complains